WebDriverAgentLib/Utilities/FBXCodeCompatibility.m

```diff
@@ -43,3 +43,28 @@ + (instancetype)fb_applicationWithPID:(pid_t)processID
 
 @end
 
+static BOOL FBShouldUseFirstMatchSelector = NO;
+static dispatch_once_t onceFirstMatchToken;
+@implementation XCUIElementQuery (FBCompatibility)
+
+- (XCUIElement *)fb_firstMatch
+{
+  SEL firstMatchSelector = NSSelectorFromString(@"firstMatch");
```

Comment:
originally firstMatch is a part of XCUIElementTypeQueryProvider protocol. Shall I add it there or it's better to do it directly in XCUIElement?